diff -ruN src/tensorflow/contrib/makefile/download_dependencies.sh src-new/tensorflow/contrib/makefile/download_dependencies.sh
--- src/tensorflow/contrib/makefile/download_dependencies.sh    2021-05-27 12:50:46.791726300 +0000
+++ src-new/tensorflow/contrib/makefile/download_dependencies.sh        2021-05-27 12:59:20.421726300 +0000
@@ -26,7 +26,7 @@
   exit 1;
 fi

-EIGEN_URL="$(grep -o 'http.*bitbucket.org/eigen/eigen/get/.*tar\.gz' "${BZL_FILE_PATH}" | grep -v mirror.bazel | head -n1)"
+EIGEN_URL="$(grep -o 'http.*bitbucket.org/eigen/eigen/get/.*tar\.gz' "${BZL_FILE_PATH}" | grep mirror.bazel | head -n1)"
 GEMMLOWP_URL="$(grep -o 'https://mirror.bazel.build/github.com/google/gemmlowp/.*zip' "${BZL_FILE_PATH}" | head -n1)"
 GOOGLETEST_URL="https://github.com/google/googletest/archive/release-1.8.0.tar.gz"
 NSYNC_URL="$(grep -o 'https://mirror.bazel.build/github.com/google/nsync/.*tar\.gz' "${BZL_FILE_PATH}" | head -n1)"
