diff --git a/inference-engine/cmake/dependencies.cmake b/inference-engine/cmake/dependencies.cmake
index 2740691..8feec66 100644
--- a/inference-engine/cmake/dependencies.cmake
+++ b/inference-engine/cmake/dependencies.cmake
@@ -146,12 +146,14 @@ if (ENABLE_OPENCV)
     elseif(LINUX)
         if (${CMAKE_SYSTEM_PROCESSOR} STREQUAL "armv7l")
             set(OPENCV_SUFFIX "debian9arm")
-        elseif (${LINUX_OS_NAME} STREQUAL "CentOS 7" OR CMAKE_CXX_COMPILER_VERSION VERSION_LESS "4.9")
+        elseif (${LINUX_OS_NAME} STREQUAL "CentOS 7" OR ${LINUX_OS_NAME} STREQUAL "Amazon 2" OR CMAKE_CXX_COMPILER_VERSION VERSION_LESS "4.9")
             set(OPENCV_SUFFIX "centos7")
         elseif (${LINUX_OS_NAME} STREQUAL "Ubuntu 16.04")
             set(OPENCV_SUFFIX "ubuntu16")
         elseif (${LINUX_OS_NAME} STREQUAL "Ubuntu 18.04")
             set(OPENCV_SUFFIX "ubuntu18")
+        elseif (${LINUX_OS_NAME} STREQUAL "Ubuntu 20.04")
+            set(OPENCV_SUFFIX "ubuntu20")
         else()
             message(FATAL_ERROR "OpenCV is not available on current platform")
         endif()
